src/git/repo.go

```diff
@@ -0,0 +1,590 @@
+package git
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/git-town/git-town/src/command"
+	"github.com/git-town/git-town/src/util"
+	"github.com/git-town/git-town/test/helpers"
+)
+
+var repoInCurrentDir *Repo
+
+// RepoInCurrentDir provides a Repo instance in the current working directory.
+func RepoInCurrentDir() *Repo {
+	if repoInCurrentDir == nil {
+		repoInCurrentDir = &Repo{
+			Dir:   ".",
+			Shell: &command.ShellInCurrentDir{},
+		}
+	}
+	return repoInCurrentDir
+}
+
+// Repo is a Git repository that exists inside a Git environment.
```

Comment:
The `Git environment` reference doesn't really make sense as Git environment is test only

Reply:
Good catch, removed.